arena-pot/src/main/scala/com/uqbar/aop/transaction/ObjectTransactionImpl.scala

```diff
@@ -206,10 +203,8 @@ class ObjectTransactionImpl(var parent: ObjectTransactionImpl, var owner: TaskOw
             try {
               ReflectionUtils.invokeSetter(keyWrapper.getKey(), k, v);
             } catch {
-              case e: RuntimeException => throw new ProgramException(e) //
-                .addInfo("Object", keyWrapper.getKey()) //
-                .addInfo("Field", k) //
-                .addInfo("Value", v);
+              case e: RuntimeException =>
```

Comment:
jajaja... sí, hoy a las 8:30 me llegaron todas las notificaciones y me puse
a leer

On Thu, Aug 11, 2016 at 11:36 AM, Fernando Dodino <notifications@github.com>
wrote:

> In arena-pot/src/main/scala/com/uqbar/aop/transaction/
> ObjectTransactionImpl.scala
> <https://github.com/uqbar-project/arena/pull/8#discussion_r74433873>:
>
> > @@ -206,10 +203,8 @@ class ObjectTransactionImpl(var parent: ObjectTransactionImpl, var owner: TaskOw
> >              try {
> >                ReflectionUtils.invokeSetter(keyWrapper.getKey(), k, v);
> >              } catch {
> > -              case e: RuntimeException => throw new ProgramException(e) //
> > -                .addInfo("Object", keyWrapper.getKey()) //
> > -                .addInfo("Field", k) //
> > -                .addInfo("Value", v);
> > +              case e: RuntimeException =>
>
> Che, no me echen la culpa. No se por qué se pusieron a ver esto ustedes,
> yo solo me puse a cerrar issues del año de la escarapela.
>
> Javi, metele pata a Wollok, lo de Arena es para el verano (argentino) del
> 2017.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/uqbar-project/arena/pull/8/files/797af34680cac9217346bc335da039d79ab65424#r74433873>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEORWIbzzrOQraMVjnAz6lOQBCoWwOIXks5qezNngaJpZM4C5ng4>
> .
>